unit-tests/run-unit-tests.py

```diff
@@ -197,34 +229,39 @@ def cat( filename ):
     except FileNotFoundError:
         error( red + testname + reset + ': executable not found! (' + exe + ')' )
         continue
+    
+    check_log(log)
+    
 
-    # Normal logs are expected to have in last line:
-    #     "All tests passed (11 assertions in 1 test case)"
-    # Tests that have failures, however, will show:
-    #     "test cases: 1 | 1 failed
-    #      assertions: 9 | 6 passed | 3 failed"
-    for ctx in grep( r'^test cases:\s*(\d+) \|\s*(\d+) (passed|failed)', log ):
-        m = ctx['match']
-        total = int(m.group(1))
-        passed = int(m.group(2))
-        if m.group(3) == 'failed':
-            # "test cases: 1 | 1 failed"
-            passed = total - passed
-        if passed < total:
-            if total == 1  or  passed == 0:
-                desc = 'failed'
-            else:
-                desc = str(total - passed) + ' of ' + str(total) + ' failed'
+# adding given directory to python search path so that python tests can find pyrealsense2.pyd
+abs = os.path.abspath(dir)
+abs = abs.replace('\\' , '/')
```

Comment:
Needs explanation... does PYTHONPATH depend on Linux directory separators? Sounds weird... and notice you also have `os.sep` which is different in Windows vs. Linux.

Make sure you test under Linux.